### PR TITLE
Advertise public ips

### PR DESCRIPTION
**Pull request by 0x5d**

### CODE REVIEW

**Comment by 0x5d** [comment]
@0xdiba please take a look!

File: README.md [hunk 1]
```diff
@@ -29,10 +29,18 @@ If already created infrastructure is going to be used, they can safely be skiped
...
         These have to be changed with according to the chosen AWS region.
         - `distro_ssh_user`: User used to ssh into the created EC2 instances.
+
 ### Required Steps: 4-6  
         
 4. Fill in the `hosts.ini` template with the user & ips, based either on the terraform output or your own infrastructure.
 5. `ansible-playbook --private-key <your_private_key> -i hosts.ini -v ansible/playbooks/provision-node.yml -e redpanda_packagecloud_token=<your_token_here> <extra variables - optional>`
-    - To start Redpanda and monitoring on the nodes, extra variable `-e start=true` can be passed to the ansible command
 
+Available Ansible variables:
```
Comment (0xdiba) [inline README.md:38]: nit: We could make this a heading (`####`).

**Review by 0xdiba — APPROVED** [review]

**Comment by senior7515** [comment]
@0x5d - were you thinking of merging this or ?

**Comment by senior7515** [comment]
@0x5d can you rebase.

**Pull request merged**